packages/ui/src/components/tailwind/ConfirmDialog/index.tsx

```diff
@@ -62,15 +64,24 @@ export const ConfirmDialog = ({ title, text, onSubmit, onClose, modalProps }: Co
         PopoverState.hidePopupover()
         onClose?.()
       }}
-      className="w-[50vw] max-w-2xl"
+      className={twMerge(
+        'w-[50vw] max-w-2xl bg-surface-1',
+        isMobile ? 'h-[90dvh] w-[50vw] min-w-[720px] max-w-2xl' : ''
+      )}
       submitLoading={modalProcessing.value}
+      rawChildren={
+        <div
+          className={twMerge(
+            'flex flex-col items-center gap-2',
+            isMobile ? 'h-[calc(90dvh-4rem-4.5rem)] justify-center' : ''
+          )}
+        >
+          <Text className="text-text-secondary">{text}</Text>
```

Comment:
same here media queries like `smh` and `lg`

Reply:
done